addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -78,6 +85,7 @@ export class Form extends Interaction {
         // TODO Translation behavior.
         this.editTranslations = false;
         // this.editTranslations = !!this.getContext(true).edit_translations;
+        this.onFieldInputDebounced = debounce(this.onFieldInput.bind(this), 400);
```

Comment:
note that it is likely unsafe => the interaction can be destroyed in the debounce delay. we need to add a check in the method to protect it

Reply:
The `debounce` has actually become absurd: the handler is empty now, it is there only to trigger the implicit `updateDom` after the event handling.